readme.md

```diff
@@ -33,20 +33,26 @@ const unusedFilename = require('unused-filename');
 
 ## API
 
-### unusedFilename(filePath)
+### unusedFilename(filePath[, incrementer])
```

Comment:
It should be an options-object, not a direct argument.

Reply:
done.